### PR TITLE
Retrieve line_section disruptions

### PR DESCRIPTION
**Pull request by dvdn**
# Description

This PR adds filter on line_section

## Issue (optional)

Issue link: http://jira.canaltp.fr/browse/BOT-42

## Pull Request type (optional)

New feature

## How to test

- Create some disruptions/impacts on line_sections and lines,
- Call api disruptions with :
    filter uri
    then filter uri + line_section=true
- Check response and retrieve your related disruptions/impacts

### CODE REVIEW

File: tests/features/list-disruptions-by-pt-object.feature [hunk 1]
```diff
@@ -318,6 +313,104 @@ Feature: list impacts by ptobject and/or uri(s)
...
+        #Query on object 'line:JDR:M12' present in a disruption
+        I fill in header "X-Contributors" with "contrib1"
+        I fill in header "X-Coverage" with "jdr"
+        I fill in header "Authorization" with "d5b0148c-36f4-443c-9818-1f2f74a00be0"
+        When I get "/disruptions?uri=line:JDR:M12"
+        Then the status code should be "200"
+        And the header "Content-Type" should be "application/json"
+        And the field "disruptions" should have a size of 1
         And the field "disruptions.0.id" should be "7ffab230-3d48-4eea-aa2c-22f8680230b6"
-        And the field "disruptions.1.id" should be "8ffab230-3d48-4eea-aa2c-22f8680230b6"
```
Comment (kadhikari) [inline tests/features/list-disruptions-by-pt-object.feature]: I don't see any test for /disruptions?uri=line:JDR:M1&line_section=true.

You can also add some tests for I don't see any test for /disruptions?uri=line:JDR:M1&line_section=false..

File: chaos/resources.py [hunk 2]
```diff
@@ -174,6 +174,7 @@ def __init__(self):
                                 action="append")
         parser_get.add_argument("current_time", type=utils.get_datetime)
         parser_get.add_argument("uri", type=str)
+        parser_get.add_argument("line_section", type=bool, default=False)
```
Comment (kadhikari) [inline chaos/resources.py]: when you use &line_section=False, i got True in debug mode.

Comment (kinnou02) [inline chaos/resources.py]: You must use `boolean`

**Comment by pthegner** [comment]
retest this please

**Pull request merged**